packages/svelte2tsx/test/svelte2tsx/index.js

```diff
@@ -16,15 +16,20 @@ describe('svelte2tsx', () => {
 		}
 
 		(solo ? it.only : it)(dir, () => {
-            const input = fs.readFileSync(`${__dirname}/samples/${dir}/input.svelte`, 'utf-8').replace(/\s+$/, '').replace(/\r\n/g, "\n");
-            const expectedOutput = fs.readFileSync(`${__dirname}/samples/${dir}/expected.tsx`, 'utf-8').replace(/\s+$/, '').replace(/\r\n/g, "\n");
+			const path = `${__dirname}/samples/${dir}`;
+            const input = fs.readFileSync(`${path}/input.svelte`, 'utf-8').replace(/\s+$/, '').replace(/\r\n/g, "\n");
+			const expectedOutput = fs.readFileSync(`${path}/expected.tsx`, 'utf-8').replace(/\s+$/, '').replace(/\r\n/g, "\n");
+			const expecterOtherOutput = fs.existsSync(`${path}/expected.js`) && require(`${path}/expected`);
 
-            const { map, code} = svelte2tsx(input, {
+            const output = svelte2tsx(input, {
 				strictMode: dir.includes('strictMode'),
 				isTsFile: dir.startsWith('ts-'),
 				filename: 'input.svelte'
 			});
-            assert.equal(code, expectedOutput);
+			assert.equal(output.code, expectedOutput);
+			if (expecterOtherOutput) {
+				expecterOtherOutput(output);
```

Comment:
I enhanced the test suite so you can test other outputs of `svelte2tsx`, see `component-events-interface-constant/expected.js` for an example.